requirements.txt

```diff
@@ -1,13 +1,14 @@
+folium==0.12.0
```

Comment:
Why not the latest 0.12.1

Reply:
https://github.com/python-visualization/folium/issues/1072 "Folium map shows fine on jupyter but does not appear on github".

Dus die cell niet runnen in de notebook

Reply:
> Why not the latest 0.12.1

Because this was the latest when I installed it? The whole requirements lists is based on package versions when we first added them.. maybe we should use >= but then you can never be sure if an update doesn't break the code.

Reply:
0.12.1 komt uit januari 2021 dus ik dacht heh dat is raar. Ja gewoon == idd

Reply:
Oh wat maf. Misschien is mijn conda outdated ofzo.